lib/score-manager.js

```diff
@@ -1,21 +1,106 @@
-/** Apply each plugin to the file, task pair and return scores. */
-function applyPlugins (plugins, file, task) {
-  let scores = {}
-
-  Object.keys(plugins).forEach((key, idx) => {
-    try {
-      scores[key] = plugins[key](file, task)
-    } catch (err) {
-      scores[key] = 'failure: ' + err.message
+const VError = require('verror').VError
```

Comment:
Wenn was schief geht dann steht da. 'Highest level error: lower level error: lowest level error'. So kann man in jedem Level des Callstacks Kontext zum Error hinzufügen, wenn es Sinn macht. Z.B. 'Invalid path: a.b.c' ist vielleicht für ein Plugin ok, bei 10 Plugins, die einen ähnlichen Path verwenden wäre besser: 'Invalid plugin configuration plugin-a: invalid path: a.b.c'. Dafür ist verror